src/tiled/tileanimationeditor.ui

```diff
@@ -20,6 +20,35 @@
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
+         <widget class="QSpinBox" name="frameTime">
+             <property name="suffix">
+              <string>ms</string>
+             </property>
+             <property name="minimum">
+              <number>20</number>
+             </property>
+             <property name="maximum">
+              <number>10000</number>
+             </property>
+             <property name="singleStep">
+              <number>20</number>
+             </property>
+             <property name="value">
+              <number>100</number>
+             </property>
+         </widget>
+
+
+     </item>
+     <item>
+         <widget class="QPushButton" name="setFrameTimeButton">
+             <property name="text">
+              <string>Set Frame Time</string>
```

Comment:
I think it would be nicer to use a label "Frame duration:", then the input box, and then the button with just "Apply" on it.